lib/vehicle_factory.rb

```diff
@@ -0,0 +1,37 @@
+class VehicleFactory
+  def initialize
+    
+  end
+
+  def create_vehicles(dmv)
+    dmv.map do |car|
+      Vehicle.new({
+        engine: :ev,
+        make:  car[:make],
+        model: car[:model],
+        vin: car[:vin_1_10],
+        year: car[:model_year]
+        })
+    end
+  end
+
```

Comment:
Really nice job using some of the other enumerables that we don't necessarily instruct on!